Solutions/Forescout Host Property Monitor/Analytic Rules/DnsniffEvent-1.yaml

```diff
@@ -0,0 +1,30 @@
+id: d272e277-f285-4dbc-ae2d-7f65ba64a79e
+name: DnsniffEvent-1
```

Comment:
Can we give a suitable name for this Analytical Rule.

Reply:
Our customers are familiar with and understand the name DnsniffEvent. And we are planning to add more examples later, so we added a number to the name.

Reply:
I understand that your customers are aware of the termologies you are using. But we are keeping in mind the interest of all the users of Sentinel. Please update the names so that they reflect the operation to be performed.

Reply:
Understand, we will update the names.